examples/zookeeper/config-and-run.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Check if conf folder is empty
+if [ ! "$(ls -A /opt/zookeeper/conf)" ]; then
+  cp /opt/zookeeper/conf.template/* /opt/zookeeper/conf/
```

Comment:
If what you want is not to over-write files that already exist, you could also `cp --no-clobber` instead of this `if` check.